samples/msal-node-samples/auth-code/index.js

```diff
@@ -71,7 +95,8 @@ const getTokenAuthCode = function (scenarioConfig, clientApplication, port) {
     });
 
     app.get("/redirect", (req, res) => {
-        const tokenRequest = { ...requestConfig.tokenRequest, code: req.query.code };
+        const tokenRequest = { ...requestConfig.tokenRequest, code: req.query.code};
+        const authCodeResponse = { nonce :req.session.nonce }
```

Comment:
Can we please add documentation links to `nonce` in the standard and in the repo?